tests/test_interestingness.py

```diff
@@ -298,6 +298,7 @@ def test_interestingness_deviation_nan():
 
     smaller_diff_score = interestingness(vis, test)
     bigger_diff_score = interestingness(vis2, test)
-    assert np.isclose(smaller_diff_score, 0.29, rtol=0.1)
-    assert np.isclose(bigger_diff_score, 0.94, rtol=0.1)
+    print(smaller_diff_score, bigger_diff_score)
+    assert np.isclose(smaller_diff_score, 0.19, rtol=0.1)
```

Comment:
I dont think we need the print statement.

Reply:
Yup I agree, we can take this out.

Reply:
Yes looks like something that was accidentally left in during some bug testing, will remove.

Reply:
I removed this from the test file.